README.md

```diff
@@ -49,6 +50,35 @@ Particle orientations are stored as
 [`scipy.spatial.transform.Rotation`](https://docs.scipy.org/doc/scipy/reference/generated/scipy.spatial.transform.Rotation.html) objects.
 These transformations should rotate the basis vectors of a reference such that they are correctly oriented in a tomogram.
 
+## `ProjectionModel`
+A `ProjectionModel` is a [pandas `DataFrame`](https://pandas.pydata.org/docs/) with specific column 
+headings for the parameters of a projection model. Together, this information constitues a 'tilt-series alignment'.
+
+| Heading         | Python name   | Semantics                                  |
+|:----------------|:--------------|:-------------------------------------------|
+| `rotation_x`    | ROTATION_X    | tomogram rotation around x-axis            |
+| `rotation_y`    | ROTATION_Y    | tomogram rotation around y-axis            |
+| `rotation_z`    | ROTATION_Z    | tomogram rotation around z-axis            |
+| `dx`            | SHIFT_X       | particle shift in x-dimension              |
+| `dy`            | SHIFT_Y       | particle shift in y-dimension              |
+| `experiment_id` | EXPERIMENT_ID | identifier for micrograph/tilt-series      |
+| `pixel_spacing` | PIXEL_SPACING | isotropic pixel/voxel spacing for shifts   |
+| `source`        | SOURCE        | reference to the file from which data came |
+
+A utility function is also provided for generating projection matrices from these data.
+
+```python
+from cryotypes.projectionmodel import projection_model_to_projection_matrices
+
+projection_matrices = projection_model_to_projection_matrices(
+    df=projection_model,  # ProjectionModel dataframe
+    tilt_image_center=(1919, 1355),  # tilt-image rotation center (xy)
+    tomogram_dimensions=(3838, 3710, 2000)  # dimensions of tomogram (xyz)
+)
+```
```

Comment:
Maybe we should provide the same for particles (if one wants to work with affines rather than pos + rot).

Reply:
possible but not sure of the benefit there, will open an issue to discuss something re this though